raven/src/main/java/com/getsentry/raven/RavenContext.java

```diff
@@ -122,4 +126,21 @@ public void recordBreadcrumb(Breadcrumb breadcrumb) {
         breadcrumbs.add(breadcrumb);
     }
 
+    /**
+     * Store the UUID of the last sent event by this thread, useful for handling user feedback.
+     *
+     * @param id UUID of the last event sent by this thread.
+     */
+    public void setLastEventId(UUID id) {
+        lastEventId = id;
+    }
+
+    /**
+     * Get the UUID of the last sent event by this thread, useful for handling user feedback.
+     *
+     * @return UUID of the last event sent by this thread.
+     */
+    public UUID getLastEventId() {
```

Comment:
Might be worth clarifying in the return type documentation that you should expect to potentially receive a null result from this method.